dialogflow_v2beta1/proto/audio_config_pb2.py

```diff
@@ -384,17 +611,29 @@
           more details.
       language_code:
           Required. The language of the supplied audio. Dialogflow does
-          not do translations. See `Language Support
-          <https://cloud.google.com/dialogflow-
-          enterprise/docs/reference/language>`__ for a list of the
-          currently supported language codes. Note that queries in the
-          same session do not necessarily need to specify the same
+          not do translations. See `Language Support <https://cloud.goog
+          le.com/dialogflow/docs/reference/language>`__ for a list of
+          the currently supported language codes. Note that queries in
+          the same session do not necessarily need to specify the same
           language.
+      enable_word_info:
```

Comment:
Add enable_word_info, speech_contexts, model_variant, single_utterance to InputAudioConfig (v2beta1)